.github/scripts/check_newsfragments.py

```diff
@@ -78,7 +76,11 @@ def check_newsfragment(fragment: Path, base: str) -> Optional[bool]:
         ret = list(
             filter(
                 lambda value: value is not None,
-                pool.map(check_newsfragment, Path("newsfragments").glob("*.rst"), args.base),
+                pool.map(
+                    check_newsfragment,
+                    Path("newsfragments").glob("*.rst"),
+                    repeat("origin/" + args.base),
```

Comment:
Wouldn't be better to include the `"origin/"` in the `args.base` when invoking the script? 

I don't think we'll need to change it, but that way we avoir the hard-coded string and we already specify "origin" in https://github.com/Scille/parsec-cloud/pull/5810/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR162

Reply:
I've chosen to add the prefix in the script because, it's require for `git log` so everytime you use that script, you need to specify `origin/`.
I find it simpler that way.

CF: the failing run that use `base` without the prefix https://github.com/Scille/parsec-cloud/actions/runs/6888409905/job/18737350209